<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>443</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>File Organizer</string>
  </property>
  <widget class="QListWidget" name="moveFolderList">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>+ Add folder (double click)</string>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>185</red>
       <green>212</green>
       <blue>195</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="infoText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="openBtn">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dirEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>211</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="renameBtn">
      <property name="text">
       <string>Rename</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="renameEdit">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="picView">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="deleteBtn">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="undoBtn">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>221</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="skipLeftBtn">
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="skipBtn">
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="doneBtn">
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>211</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="previewBtn">
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fileNameLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sample file.txt</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
